sites/all/modules/custom/chabadsuite/cs_user_permission/cs_user_permission.features.user_permission.inc

```diff
@@ -742,7 +720,9 @@ function cs_user_permission_user_default_permissions() {
   // Exported permission: 'administer permissions'.
   $permissions['administer permissions'] = array(
     'name' => 'administer permissions',
-    'roles' => array(),
+    'roles' => array(
+      'SUPER ADMIN' => 'SUPER ADMIN',
```

Comment:
@yosefromano I added this - I assume it wasn't left out on purpose? Just found I couldn't edit permissions, even with super admin role